source/documentation/virtual_machine_management_guide/topics/Live_migration_prerequisites.adoc

```diff
@@ -66,7 +66,7 @@ See link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.
 +
 [NOTE]
 ====
-The virtual machine will have three interfaces available, in order for migration to work properly, the main interface should be activated and used for connection. The main interface acts as a master for the remaining two interfaces.
+The virtual machine will have three interfaces available: a main interface and two secondary interfaces controlled by it. In order for migration to succeed, activate the main interface and use it for connection.
 ====
```

Comment:
I considered "connection purposes" but chose brevity. WDYT?

Reply:
Brevity is good!

Couple small nits. ISG says to use present tense. I like the way you eliminated 'master'. I think we could make it a bit more concise.

Suggestion: 

The virtual machine has three network interfaces: a primary interface and two secondary interfaces. The primary interface must be active and connected in order for migration to succeed.

I don't think you need to say "controlled by it". That is implied by "primary" and "secondary".

Reply:
I know that line 54 is not part of your changes, but could you remove the period from the end of "=== Configuring Virtual Machines with SR-IOV-Enabled vNICs with minimal downtime."? It's really visible in the published doc.